packages/db/types/stub.d.ts

```diff
@@ -2,59 +2,47 @@
 // graphql typescript definitions
 
 declare namespace DataModel {
-  type IResourceReferenceInput = any;
-  type IBytecode = any;
-  type IBytecodeInput = any;
-  type IBytecodesAddInput = any;
-  type IBytecodesAddPayload = any;
-  type ICompilation = any;
-  type ICompilationInput = any;
-  type IProcessedSourceInput = any;
-  type ISourceMapInput = any;
-  type ICompilationsAddInput = any;
-  type ICompilationsAddPayload = any;
-  type ICompiler = any;
-  type ICompilerInput = any;
-  type IContract = any;
-  type IContractInput = any;
-  type IContractInstance = any;
-  type IContractInstanceInput = any;
-  type IContractInstancesAddInput = any;
-  type IContractInstancesAddPayload = any;
-  type IContractsAddInput = any;
-  type IContractsAddPayload = any;
-  type IInstruction = any;
-  type INameRecord = any;
-  type INameRecordInput = any;
-  type INameRecordsAddInput = any;
-  type INameRecordsAddPayload = any;
-  type INetwork = any;
-  type INetworkInput = any;
-  type INetworksAddInput = any;
-  type IProject = any;
-  type IProjectInput = any;
-  type IProjectsAddInput = any;
-  type IProjectsAddPayload = any;
-  type IProjectName = any;
-  type IProjectNameInput = any;
-  type IProjectNamesAssignPayload = any;
-  type ISource = any;
-  type ISourceInput = any;
-  type ISourceRange = any;
-  type ISourcesAddInput = any;
-  type ISourcesAddPayload = any;
-
-  interface IWorkspaceQuery {
```

Comment:
Did you remove these interfaces on purpose? Just checking!

Reply:
Yep - they were unused. Probably should've done that separately but meh